mwdb/web/src/components/Settings/Views/UsersPendingList.js

```diff
@@ -11,28 +12,17 @@ import {
 
 export default function UsersPendingList() {
     const viewAlert = useViewAlert();
-    const [pendingUsers, setPendingUsers] = useState([]);
+    const config = useContext(ConfigContext);
```

Comment:
Let's destructure that configuration and use only the `getPendingUsers` callback as dependency.
```suggestion
    const {getPendingUsers}  = useContext(ConfigContext);
```